narwhal/primary/src/proposer.rs

```diff
@@ -228,6 +228,24 @@ impl Proposer {
         )
         .await;
 
+        let leader_support = if this_round % 2 == 0 {
+            "even_round"
```

Comment:
Maybe you could also use two labels here, if this is the leader of the round it could be `leader`, otherwise `even_round`